frontend/src/components/dashboard/infoAlert.js

```diff
@@ -0,0 +1,29 @@
+import { Alert } from 'reactstrap'
+import cookie from 'js-cookie'
+import { useState } from 'react'
+
+const InfoAlertName = 'INFO-ALERT'
+const AlertOff = 'FALL-2019-UPDATE' // change value if you want to trigger a new alert
+
```

Comment:
Minor: I feel like this is a good thing to have as a component that takes the expiration and inner JSX as a prop—I think it's something that could be useful to have as a wrapping component around `Alert` rather than having the infoAlert be hardcoded to use the specific "Fall 2019 Updates"

Reply:
Yes, my original thought was that we should only use it for the dashboard, which was why it was done this way. I've updated it for us to have a general component for alerts with the cookie functionality and added a specific component for the dashboard alert. The more general component `infoAlert.js` now requires a bit more understanding on how it works for someone to actually use it though. 

Now, the point of this component (now moved to `updateAlert.js` is for us to put in some alert (say for updates) - for now, it needs to committed into the code since we don't have configuration management. But afterward, what if we want to provide updates for Spring 2020?   Then, we need to update the `AlertOff` value to be another value so the alert actually shows up.